docs/modules/getting-started/pages/metrics.adoc

```diff
@@ -0,0 +1,69 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use and integrate the AeroGear Metrics SDK on a Cordova application.
+
+=== Install Dependencies
+
+Install the necessary dependencies in the app by running the following commands in your terminal:
+
+[source.bash]
+npm install @aerogear/app @aerogear/core
+cordova plugin add @aerogear/cordova-plugin-aerogear-metrics
+
+=== Deploy Metrics in Openshift
+
+Deploy an instance of Metrics from the Openshift console.
+
+=== Get the App Configuration
+
+The configuration needed to initialize the AeroGear SDK is contained in a JSON file generated by the https://github.com/aerogear/mobile-cli[AeroGear Mobile CLI]. It has the necessary parameters for every deployed service as well as other global ones. It should look like:
+
+```
+{
+  "version": 1,
+  "clusterName": "192.168.64.74:8443",
+  "namespace": "my-project",
+  "clientId": "my-app-id",
+  "services": [
+    {
+      "id": "metrics",
+      "url": "http://...",
+      ...
+    },
+    ...
+  ]
+}
+```
+
+=== Usage
+
+After dependencies are installed, call `init` from `@aerogear/app` when the app just starts:
+
+```
+import { init } from "@aerogear/app";
+
+const appConfig = { ... };
+
+document.addEventListener("deviceready", () => init(appConfig), false);
+```
+
+As part of the SDK initialization, the relevant app and device data will be sent to the server automatically. Metrics is designed to send the data only on app startup, hence it shouldn't (and can't) be used manually.
+
+=== Provided Metrics
```

Comment:
IMHO this duplicates existing metrics documentation in other PR. Best to just have links here to other inidividual SDK's.